### PR TITLE
edits to backup info

### PR DESCRIPTION
**Pull request by fiquick**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-328.surge.sh

Updated pages:
- [Backup, export and restore](https://neo4j-docs-aura-328.surge.sh/aura/auradb/managing-databases/backup-restore-export)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-328.surge.sh

Updated pages:
- [Backup, export and restore](https://neo4j-docs-aura-328.surge.sh/aura/auradb/managing-databases/backup-restore-export)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-328.surge.sh

Updated pages:
- [Backup, export and restore](https://neo4j-docs-aura-328.surge.sh/aura/auradb/managing-databases/backup-restore-export)

File: modules/ROOT/pages/auradb/managing-databases/backup-restore-export.adoc [hunk 1]
```diff
@@ -32,6 +32,12 @@ A *Scheduled* snapshot is a snapshot that is automatically triggered when you fi
 Scheduled snapshots are run automatically once a day for Professional instances and once an hour for Enterprise instances. 
 Only one scheduled snapshot is retained per day after 7 days for Enterprise instances.
 
+[NOTE]
+====
+For AuraDB Enterprise database instances running Neo4j v4, from day 0 to 7 scheduled snapshots run automatically once every 6 hours.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/auradb/managing-databases/backup-restore-export.adoc:37]:
```suggestion
For AuraDB Enterprise database instances running Neo4j v4.x, from day 0 to 7 scheduled snapshots run automatically once every 6 hours. 
```

**Review by AlexicaWright — COMMENTED** [review]
Just a small suggestion for version 4.x .

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-328.surge.sh

Updated pages:
- [Backup, export and restore](https://neo4j-docs-aura-328.surge.sh/aura/auradb/managing-databases/backup-restore-export)

**Review by AlexicaWright — APPROVED** [review]
LGTM 👍 , thanks @fiquick !

**Review by AlexicaWright — APPROVED** [review]
👍

**Comment by fiquick** [comment]
[Aura Enterprise Backup Changes](https://trello.com/c/Qpcg6yaL/306-aura-enterprise-backup-changes)

**Pull request merged**